lazyflow/operatorWrapper.py

```diff
@@ -90,7 +90,7 @@ def __init__(
 
         self._initialized = False
 
-        if operatorClass.name == "":
+        if operatorClass.name == Operator.name:
             self._name = "Wrapped " + operatorClass.__name__
         else:
             self._name = "Wrapped " + operatorClass.name
```

Comment:
The default naming of operators gives their _instances_ the class name as their default `self.name`, but the _class_ remains with `cls.name == "Operator (base class)`. This branching runs into both cases a lot because we have many with and many without name in the class definition. Now that I think about it... we already have a custom OperatorMetaClass. This is exactly the kind of stuff metaclasses are for :smile: I will see if I can move this into the meta class, as I think that will allow us to basically remove all the other default renaming.

The wrapper has `._name` because `.name` is a property, which it is so that the setter can track whether it's been called using `._customName`, which exists so that the wrapper can take the `instance.name` from the wrapped operator once it creates instances of it.

The switch in `wrapper.__init__` is necessary as long as operator default naming only happens to instances, but I don't see the point of renaming the wrapper when it creates instances of the wrapped op. The only benefit of this extra logic is to catch if the custom `self.name` is set inside `MyOp.__init__`. Which none of our operators do. I will remove the renaming on creating wrapped instances, together with `_customName`, `_name` and the `name` getter/setter.

Overall the value of all this logic is pretty limited anyway, we have maybe 5 or 6 custom names in our entire code base. Those few cases where it really seems to help could also be covered with creating an empty subclass with a unique name...